aptos-move/e2e-tests/src/executor.rs

```diff
@@ -653,15 +666,17 @@ impl FakeExecutor {
         self.block_time / 1_000_000
     }
 
-    //// exec_module is like exec(), however, we can run a Module published under
-    //// the creator address instead of 0x1, as what is currently done in exec.
+    /// exec_module is like exec(), however, we can run a Module published under
+    /// the creator address instead of 0x1, as what is currently done in exec.
```

Comment:
Let's change the name here since the function has specialized in measuring the running time.

Reply:
Renamed it to `exec_module_record_running_time`